.github/workflows/release.yml

```diff
@@ -118,38 +117,12 @@ jobs:
           echo "Publish docs to Confluence: $PUBLISH_DOCS"
           echo "Append notes to Confluence: $APPEND_NOTES"
           echo "Commit details: $COMMIT_DETAILS"
-          echo github.event_name
```

Comment:
- Consider validating the response from the Confluence API in the `Append Fix Notes to Confluence` step to handle potential errors.